AndroidSDKCore/src/main/java/com/leanplum/Leanplum.java

```diff
@@ -1941,6 +1961,12 @@ public void response(JSONObject response) {
               if (response.optBoolean(Constants.Keys.LOGGING_ENABLED, false)) {
                 Constants.loggingEnabled = true;
               }
+
+              Map<String, Object> variantDebugInfo = JsonConverter.mapFromJsonOrDefault(
```

Comment:
It seems that this is duplicated from `startHelper`.  Any good way to avoid duplicating same code?